app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    like = current_user.likes.new(
+      author_id: current_user.id,
+      post_id: @post.id
+    )
+    if like.save
+      redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Success!'
+    else
+      redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", alert: 'Error occured!'
+    end
+  end
```

Comment:
✅ A user should be able to add a like to posts 👍🏻 

This can be refactored to

```ruby
def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
```